HelpSource/Classes/APF.schelp

```diff
@@ -1,37 +1,30 @@
-class:: APF
-summary:: FIXME: APF purpose.
-categories::  UGens>Filters>Linear, UGens>Undocumented
+title:: APF
+summary:: FIXME: APF purpose
+categories:: UGens>Filters>Linear, UGens>Undocumented
 
 
-Description::
+description::
+
 
 FIXME: APF description.
 
 
 classmethods::
 
+
 method::ar, kr
 
 argument::in
-
 FIXME: explain parameter in.
 
-
 argument::freq
-
 FIXME: explain parameter freq.
 
-
 argument::radius
-
 FIXME: explain parameter radius.
 
-
 argument::mul
-
-Output will be multiplied by this value.
-
+output will be multiplied by this value.
```

Comment:
One general thing: since these are whole sentences, they normally start with a capital letter. Of course we are by far not consistent with this. I'll ignore this in further down parts.

The best would be to use small letters for expressions that are not full sentences, and capital letter for sentences.

I don't mind too much, but I remember there were people who took time to change them all to capital letters in the past! ^_^

Reply:
First, I tried to follow the general convention you mentioned:
- Phrasal expressions start with lower case.
- Complete sentences are capitalised.

Unfortunately, I did not configure a good regexpression code to do this, so I lowered all letters after `summary::`, `returns::`, `method::` and `argument::` to lowercase, using `^(summary:: |returns::\n\n|method::.*\n\n|argument::.*n)`. 
From the second sentence on, I did not change the first letters. See No. 19. in https://github.com/supercollider/supercollider/pull/6534#issuecomment-2612920068

The easiest way is to capitalise all the first letters after `summary::` , `returns::`, `method::` and `argument::`. It will only take about 30-40 minutes manually with VSCodium using the above regular expression code.